packages/react-magma-dom/src/components/Select/Select.stories.tsx

```diff
@@ -61,6 +70,60 @@ export const Multi = (props: MultiSelectProps<SelectOptions>) => (
   />
 );
 
+const helpLinkLabel = 'Learn more';
+const onHelpLinkClick = () => {
+  alert('Help link clicked!');
+};
+
+export const WithContent = (props: SelectProps<SelectOptions>) => (
+  <Select
+    additionalContent={
+      <Tooltip content={helpLinkLabel}>
+        <IconButton
+          aria-label={helpLinkLabel}
+          icon={<HelpIcon />}
+          onClick={onHelpLinkClick}
+          type={ButtonType.button}
+          size={ButtonSize.small}
+          variant={ButtonVariant.link}
+        />
+      </Tooltip>
+    }
+    labelText="Helper icon"
+    {...props}
+    items={[
+      { label: 'Red', value: 'red' },
+      { label: 'Blue', value: 'blue' },
+      { label: 'Green', value: 'green' },
+    ]}
+  />
+);
+
+export const WithContentMulti = (props: MultiSelectProps<SelectOptions>) => (
+  <Select
+    additionalContent={
+      <Tooltip content={helpLinkLabel}>
+        <IconButton
+          aria-label={helpLinkLabel}
+          icon={<HelpIcon />}
+          onClick={onHelpLinkClick}
+          type={ButtonType.button}
+          size={ButtonSize.small}
+          variant={ButtonVariant.link}
+        />
+      </Tooltip>
+    }
+    isMulti
+    labelText="Helper icon"
+    {...props}
+    items={[
+      { label: 'Red', value: 'red' },
+      { label: 'Blue', value: 'blue' },
+      { label: 'Green', value: 'green' },
+    ]}
+  />
+);
```

Comment:
Instead of a whole story for Multi with content, can we make `isMulti` an toggle for the `WithContent` story?